datafusion/sqllogictest/test_files/parquet.slt

```diff
@@ -169,6 +171,38 @@ SELECT min(date_col) FROM test_table;
 ----
 1970-01-02
 
+# Clean up
+statement ok
+DROP TABLE test_table;
+
+# Do one more test, but order by numeric columns:
+# This is to exercise file group sorting, which uses file-level statistics
+# DataFusion doesn't currently support string column statistics
+statement ok
+CREATE EXTERNAL TABLE test_table (
+  int_col INT NOT NULL,
+  string_col TEXT NOT NULL,
+  bigint_col BIGINT NOT NULL,
+  date_col DATE NOT NULL
+)
+STORED AS PARQUET
+WITH HEADER ROW
+WITH ORDER (int_col ASC NULLS LAST, bigint_col ASC NULLS LAST)
+LOCATION 'test_files/scratch/parquet/test_table';
+
+# Check output plan again, expect an "output_ordering" clause in the physical_plan -> ParquetExec:
+# After https://github.com/apache/arrow-datafusion/pull/9593 this should not require a sort.
```

Comment:
I don't fully understand this comment -- this is a new test added as part of that PR, right?

Reply:
Yeah I guess the wording is confusing since it implies the test existed before this PR. Either way I'm going to make a new sqllogictest so I'll rewrite this comment